docs/multicluster/user-guide.md

```diff
@@ -503,6 +508,102 @@ Events:
   Warning ACNPImportFailed     2m11s  resourceimport-controller  ACNP Tier random does not exist in the importing cluster test-cluster-west
 ```
 
+## Antrea Multi-cluster Deployment with Service Connectivity
```

Comment:
I assume Gateway is a must for MCS since Antrea 1.7? In this case, this section should be moved to "Antrea Multi-cluster Installation".

Reply:
I think we need a topology diagram in this doc, which talks about three clusters forming a ClusterSet, with one to be both leader and member. And we should add Service CIDR and Gateway IPs for each cluster. And based on the information, we document the steps to create a ClusterSet with Gateways.